salt/modules/virt.py

```diff
@@ -1136,6 +1139,38 @@ def _gen_xml(
         context["disks"].append(disk_context)
     context["nics"] = nicp
 
+    # Process host devices passthrough
+    hostdev_context = []
+    for hostdev_name in host_devices or []:
+        hostdevice = conn.nodeDeviceLookupByName(hostdev_name)
```

Comment:
yes it can error out. One of its uses is in a try/except block catching libvirt errors... which is fine. The other doesn't. I'll add a try/except at this level too then.